### PR TITLE
text-to-speech

### PR DESCRIPTION
**Pull request by Bahare09**
-create TextToSpeech.js
-create   TextToSpeech.css
-updare Output.js

### CODE REVIEW

File: client/src/components/Input.js [hunk 1]
```diff
@@ -50,9 +49,7 @@ const Input = ({ onSubmit }) => {
 			<button type="submit" className="submit-button" onClick={handleSubmit}>
 				&#10148;
 			</button>
-			<div className="text-to-speech-input">
-				<TextToSpeechButton />
-			</div>
+			<div className="text-to-speech-input"></div>
```
Comment (DomVinyard) [inline client/src/components/Input.js:52]:
```suggestion
			<div className="text-to-speech-input"></div>
```

File: client/src/components/InputOutput.css [hunk 2]
```diff
@@ -1,153 +1,153 @@
...
-  }
-
-  /* ------------ Responsive input & output ------------ */
-
-  .input-output-wrapper {
-    display: flex;
-    width: 100%;
-    /* gap: 5px; */
-    /* margin-top: 40px; */
-    /* margin-left: 20px; */
```
Comment (DomVinyard) [inline client/src/components/InputOutput.css:125]: delete commented out code

File: client/src/components/InputOutput.css [hunk 3]
```diff
@@ -1,153 +1,153 @@
...
+}
+
+/* ------------ Responsive input & output ------------ */
+
+.input-output-wrapper {
+	display: flex;
+	width: 100%;
+	/* gap: 5px; */
+	/* margin-top: 40px; */
+	/* margin-left: 20px; */
```
Comment (DomVinyard) [inline client/src/components/InputOutput.css:124]: delete

**Review by DomVinyard — APPROVED** [review]
looks great

**Review by hadipournigjeh — APPROVED** [review]

**Pull request merged**